hack/local-up-karmada.sh

```diff
@@ -105,7 +106,14 @@ util::create_cluster "${PULL_MODE_CLUSTER_NAME}" "${MEMBER_CLUSTER_KUBECONFIG}"
 #step2. make images and get karmadactl
 export VERSION="latest"
 export REGISTRY="swr.ap-southeast-1.myhuaweicloud.com/karmada"
-make images GOOS="linux" --directory="${REPO_ROOT}"
+make image-karmada-aggregated-apiserver \
+      image-karmada-controller-manager \
+      image-karmada-scheduler \
+      image-karmada-descheduler \
+      image-karmada-webhook  \
+      image-karmada-scheduler-estimator \
+      image-karmada-interpreter-webhook-example GOOS="linux" --directory="${REPO_ROOT}"
```

Comment:
Now we maintain ht component list in several places. Consider merge it. It can be done in later PR.

https://github.com/karmada-io/karmada/blob/84fe09e344785c8345081952fc1bcab2892f7f29/Makefile#L24-L31

https://github.com/karmada-io/karmada/blob/84fe09e344785c8345081952fc1bcab2892f7f29/hack/util.sh#L24-L35

Reply:
Agree. these should in one place